EngineLayer/Mods/Mods.txt

```diff
@@ -5,19 +5,22 @@ TG   K or N
 PP   Anywhere.
 CF   O1
 MT   Common Biological
+DR   Unimod; 35.
 //
 ID   Carboxylation
 TG   K or D or E
 PP   Anywhere.
 MT   Common Biological
 CF   C1 O2
+DR   Unimod; 299.
 //
 ID   Phosphorylation
 TG   S or T
 PP   Anywhere.
 NL   HCD:H0 or HCD:H3 O4 P1
 MT   Common Biological
 CF   H1 O3 P1
+DR   Unimod; 21.
 //
 ID   Phosphorylation
```

Comment:
do you know why tyrosine is separate?

Reply:
tyrosine has a diagnostic ion while S/T don't